test/RelayServer.test.ts

```diff
@@ -1242,6 +1260,51 @@ contract('RelayServer', function (accounts) {
     // })
   })
 
+  describe('Alerted state as griefing mitigation', function () {
+    let newServer: RelayServer
+    beforeEach('should enter an alerted state for a configured blocks delay after paymaster rejecting an on-chain tx', async function () {
+      id = (await snapshot()).result
+      newServer = await bringUpNewRelay({ alertedBlockDelay: 100 })
+      await newServer._worker(await _web3.eth.getBlock('latest'))
+      await attackTheServer(newServer)
+    })
+    afterEach(async function () {
+      await revert(id)
+    })
+
+    async function attackTheServer (server: RelayServer): Promise<void> {
+      const _sendTransactionOrig = server._sendTransaction
```

Comment:
JS question: as far as I understand, this method will no longer have access to 'this' because it is unbound, is it correct? This is why you pass 'server' to the 'call'?

Reply:
Yes, you're correct.